cli.py

```diff
@@ -61,6 +67,27 @@ def load_organization(env: str, organization_id: str) -> Organization | None:
     return organization
 
 
+@cli
+def update_organizations(env: str = "demo"):
+    """Refresh and complement organizations"""
+    print("Updating organizations...")
+    organizations = app.session.query(Organization).all()
+    custom_organizations = load_custom_organizations(env)
+    for organization in organizations:
+        fresh_organization = load_organization(env, organization.organization_id, refresh=True)
+        if not fresh_organization:
+            continue
```

Comment:
Obsolete org -> delete ?

Reply:
Nope, it might still be linked to deleted datasets. We should handle a `deleted: bool` attribute on Organization too.

Reply:
https://github.com/ecolabdata/ecospheres-dashboard-backend/issues/30